### ISSUE TITLE
flask-cors flask-restx install error

### ISSUE THREAD

**Issue by Wninayyds** [schutzwerk/PROBoter#1]
![e204b1c4da84b59caac95332c824df6](https://github.com/schutzwerk/PROBoter/assets/90488923/6698272c-731f-418f-b209-b17ad0ae417b)
look ！
hi 
Is pip install flask-corsflask-restx written wrong?
Because it should be pip install flask-cors flask-restx

**Comment by tqamarVT** [comment]
Yes, in the setup.py file under the pcb-analysis directory, there is a typo.

    install_requires=[
        'werkzeug==2.1.2',
        'flask',
        'flask-cors'
        'flask-restx',
        'tensorflow',
        'opencv-python-headless',
        'pytesseract',
    ],

There needs to be a comma after 'flask-cors'
